### ISSUE TITLE
Nav block list view - "Add block" is announced twice when focusing the "appender".

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#48019]
"Add block" is announced twice when focusing the "appender".

_Originally posted by @getdave in https://github.com/WordPress/gutenberg/issues/46939#issuecomment-1427668766_

**Comment by alexstine** [comment]
I gave this another test and cannot actually prevent this from happening. I think this is just a screen reader issue with ARIA. The `aria-label` is applied to the button but somehow, the `tr` also gets the same `aria-label` read. Screen reader must be looking for context and finding the lower label on the button element. Closing this one.
